### PR TITLE
new update

### PR DESCRIPTION
**Pull request by penguin011**
index.html file is changed few.

### CODE REVIEW

**Review by penguin011 — COMMENTED** [review]
I accept.

**Comment by globalkitty** [comment]
I want best code.

**Review by globalkitty — CHANGES_REQUESTED** [review]
i have best.

**Review by anochkasky — APPROVED** [review]
I accept.

**Comment by penguin011** [comment]
Thanks

File: index.html
Comment (anochkasky) [inline index.html:1]: That's okay.

**Review by anochkasky — APPROVED** [review]
ok

**Review by anochkasky — APPROVED** [review]
accepted

**Review by anochkasky — COMMENTED** [review]
ok

**Pull request merged**